keps/0019-package-api-versioning.md

```diff
@@ -101,7 +411,14 @@ To avoid disambiguities, the `appVersion` of a package will be part of the tarba
 
 Because existing packages already set an `apiVersion` in their `operator.yaml` and the described change only affects repository indexes, this doesn't break any existing packages. Older versions of KUDO will still work with the new indexes because the additional `apiVersion` field isn't used when parsing the old `Metadata` struct.
 
+## Open Questions
+
+- It should be possible to specify a set of versions that a package can upgrade to or downgrade from. Where do we store this information?
```

Comment:
should this maybe be another KEP? It feels like it's getting pretty big this one

Reply:
Agree. Will add to non-goals.